### PR TITLE
Add aliases for Corerev >= 16, Corerev >= 22, and 802.11ac

### PR DESCRIPTION
**Pull request by C0FFEEC0FFEE**
I added new aliases according to d11.h version 578947 2015-08-13 04:46:06Z [1].

[1] http://github.com/tuapuikia/asuswrt-phantom/blob/master/release/src-rt-7.14.114.x/src/include/d11.h

### CODE REVIEW

**Comment by mbuesch** [comment]
You are creating quite some new defines that are already defined with a different (mostly generic) name. Please remove the old defines before adding new ones.

**Comment by C0FFEEC0FFEE** [comment]
I removed the old defines.

**Comment by mbuesch** [comment]
Can you squash the two commits into one? That's easier to review.

**Comment by C0FFEEC0FFEE** [comment]
I squashed them.

File: debug/include/shm.inc [hunk 1]
```diff
@@ -65,11 +64,20 @@
...
+
+// New SHM addresses; Source: d11.h 578947 2015-08-13 04:46:06Z
+// http://github.com/tuapuikia/asuswrt-phantom/blob/master/release/src-rt-7.14.114.x/src/include/d11.h
+/* Host flags to turn on ucode options */
+#define	SHM_HOST_FLAGS1		SHM(0x05E)
+#define	SHM_HOST_FLAGS2		SHM(0x060)
+#define	SHM_HOST_FLAGS3		SHM(0x062)
+#define	SHM_HOST_FLAGS4		SHM(0x078)
+#define	SHM_HOST_FLAGS5		SHM(0x0D4)
+
```
Comment (mbuesch) [inline debug/include/shm.inc]: We already have bit definitions for the values of these fields. See SHM_HF_... definitions above.
Please just rename these to SHM_HOST_FLAGS1-5 and just add 4/5 without bit definitions.

File: debug/include/spr.inc [hunk 2]
```diff
@@ -154,30 +154,41 @@
...
-#define SPR_TXE0_0x12				spr089
-#define SPR_TXE0_0x14				spr08a
 #define SPR_TXE0_0x16				spr08b
+
+// New Registers; Source: d11.h 578947 2015-08-13 04:46:06Z
+// http://github.com/tuapuikia/asuswrt-phantom/blob/master/release/src-rt-7.14.114.x/src/include/d11.h
+#define SPR_TXE0_MMPLCP0        spr088
+#define SPR_TXE0_MMPLCP1        spr089
+#define SPR_TXE0_PHY_CTL1       spr08a
+
```
Comment (mbuesch) [inline debug/include/spr.inc]: Please reorganize this to keep the ascending ordering of the spr numbers.

File: debug/include/spr.inc [hunk 3]
```diff
@@ -657,6 +814,37 @@
...
+//-----  data : Type of Bits (2, 3) and Subtype of Bits (4, 5, 6, 7)
+#define MAC_SUBTYPE_DATA                    0x08
+#define MAC_SUBTYPE_DATA_CFACK              0x18
+#define MAC_SUBTYPE_DATA_CFPOLL             0x28
+#define MAC_SUBTYPE_DATA_CFACK_CFPOLL       0x38
+#define MAC_SUBTYPE_DATA_NULL               0x48
+#define MAC_SUBTYPE_DATA_CFACK_NULL         0x58
+#define MAC_SUBTYPE_DATA_CFPOLL_NULL        0x68
+#define MAC_SUBTYPE_DATA_CFACK_CFPOLL_NULL  0x78
+
```
Comment (mbuesch) [inline debug/include/spr.inc]: Why do we need these SUBTYPE definitions here? These are not Broadcom hardware specific.

File: debug/include/shm.inc (hunk 1 above)
Comment (C0FFEEC0FFEE) [inline debug/include/shm.inc]: I fixed this in commit 122ca374c577e8bb2a623161c2ad643e4abe54f6.

File: debug/include/spr.inc (hunk 2 above)
Comment (C0FFEEC0FFEE) [inline debug/include/spr.inc]: I fixed this in commit 32c2a3c7c9726592acc21afd73541ba1d3b53b72.

File: debug/include/spr.inc (hunk 3 above)
Comment (C0FFEEC0FFEE) [inline debug/include/spr.inc]: I fixed this in commit f7016b5b2482f47b639ccd6ae4a0e8cf3d767be1.

**Pull request merged**